backend/Dockerfile

```diff
@@ -48,9 +49,25 @@ RUN apt-get -y update && apt-get install -y \
     libc6-dev-i386 \
     && rm -rf /var/lib/apt/lists/*
 
-COPY requirements.txt /backend/requirements.txt
 
-RUN python -m pip install -r /backend/requirements.txt --no-cache-dir
+ENV WINEPREFIX=/tmp/wine
+
+# create a non-root user & /sandbox with correct ownership
+RUN useradd --create-home user \
+    && mkdir -p /sandbox \
+    && chown -R user:user /sandbox \
+    && mkdir -p "${WINEPREFIX}" \
+    && chown user:user "${WINEPREFIX}"
+
+COPY pyproject.toml /backend/pyproject.toml
+COPY poetry.lock /backend/poetry.lock
+
+# Install poetry and dependencies
+USER user
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
```

Comment:
Where are the "poetry" deps installed to? the `/home/user/.<something>`...

trying to decide whether this should all be done in the docker image (as it was with requirements.txt) or as part of entrypoint.sh (as the frontend entrypoint does -- but node_modules are downloaded into /frontend so persist)

Reply:
I actually don't know... shorry